pynblint/nb_linting.py

```diff
@@ -452,6 +452,25 @@ def is_titled(notebook):
         return True
 
 
+def is_not_copy(notebook):
+    """
+    The function takes a notebook and checks whether it is a copy of another notebook: "notebook-Copy1.ipynb"
+    Args:
+        notebook(Notebook): python object representing the notebook
+
+    Returns:
+        boolean: False if the name of the notebook end in "-Copy<n>.ipynb", True otherwise
+
+    A way you might use me is
+
+    copy = is_not_copy(notebook)
+    """
+    if "-Copy" not in os.path.basename(notebook.path):
+        return True
+    else:
+        return False
```

Comment:
È questa la logica che abbiamo seguito anche per gli altri controlli?

Io immagino che un check debba restituire `True` se viene trovato il relativo difetto e `False` altrimenti.

Per esempio, guardando nel notebook, il check per i notebook senza titolo dà esito `True` se il difetto sussiste e `False` altrimenti. È bene restare coerenti, altrimenti all'aumentare dei check non si capisce più nulla.

Reply:
@louieQ per quanto riguarda le funzioni di linting che restituiscono un booleano, restituiscono True se si è seguita la best practice, false altrimenti. Alcuni esempi:

- isFilenameCharsetRestricted
- isTitled

in questo caso la best practice è quella della non-presenza di notebook duplicati, quindi credo che si rimanga coerenti con le scelte fatte